context/src/main/java/io/micronaut/runtime/converters/time/TimeConverterRegistrar.java

```diff
@@ -81,6 +82,7 @@ public class TimeConverterRegistrar implements TypeConverterRegistrar {
     private static final Pattern DURATION_MATCHER = Pattern.compile("^(-?\\d+)([unsmhd])(s?)$");
     private static final int MILLIS = 3;
 
+    @NextMajorVersion("Consider deletion of LocalDate and LocalDateTime converters")
```

Comment:
whats the reason?

Reply:
There is a forgotten TODO, I have added the annotation so it will be easier to find